src/nft/components/collection/Card.css.ts

```diff
@@ -20,7 +20,23 @@ export const loadingBackground = style({
   background: `linear-gradient(270deg, ${themeVars.colors.backgroundOutline} 0%, ${themeVars.colors.backgroundSurface} 100%)`,
 })
 
-export const notSelectedCard = style([card])
+export const notSelectedCard = style([
+  card,
+  {
+    ':after': {
+      content: '',
+      position: 'absolute',
+      top: '0px',
+      right: ' 0px',
+      bottom: ' 0px',
+      left: '0px',
+      border: ' 1px solid',
+      borderRadius: '16px',
+      borderColor: '#5D678524',
+      pointerEvents: 'none',
+    },
```

Comment:
most of the styles in this pseudo element can be added to `card` or their own shared style instead of being defined twice